scripts/index.js

```diff
@@ -0,0 +1,48 @@
+require('dotenv').config()
```

Comment:
can you move that to scripts/lib/bigquery

Reply:
yeah sure